portfolio/bin/src/main/webapp/style.css

```diff
@@ -0,0 +1,26 @@
+#content {
+    margin-left: auto;
```

Comment:
Please also change the beginning space from 4 to 2 for this file.

Reply:
fixed